### PR TITLE
feat: update wordpress template

### LINKED ISSUE #558: Model: Add Kusion Template for WordPress Application with Local Database
### What would you like to be added?

An online template for WordPress Application with local database should be added. 

### Why is this needed?

Users can get started with Kusion more quickly and convenient using the template KCL files we are providing.

### PR DESCRIPTION
**Pull request by liu-hm19**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

fix https://github.com/KusionStack/kusion/issues/558
#### 2. What is the scope of this PR (e.g. component or file name):

/wordpress
#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other
This PR updates the wordpress template with local database and cloud rds. 

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [ ] N
- [x] Y 

This PR changes the origin wordpress template into wordpress-cloud-rds and adds a new template with local database. 
#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [x] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: wordpress-cloud-rds/dev/platform.k [hunk 1]
```diff
@@ -3,12 +3,13 @@ import catalog.models.schema.v1 as ac
...
+        privateRouting = False
+
         # SubnetID defines the virtual subnet ID associated with the VPC that the rds
         # instance will be created in. The rds instance won't be created in user's own VPC
-        # if this field is not provided.
-        subnetID = [your-subnet-id]
-        securityIPs = ["0.0.0.0/0"]
-        privateRouting = False
+        # but in default VPC of cloud vendor, if this field is not provided.
+        # subnetID = [your-subnet-id]
```
Comment (SparkYuan) [inline wordpress-cloud-rds/dev/platform.k:11]: Uncomment this field as it is necessary. Users cannot succeed without it.

File: wordpress-cloud-rds/dev/platform.k [hunk 2]
```diff
@@ -3,12 +3,13 @@ import catalog.models.schema.v1 as ac
...
+        
+        # category = "serverless_basic"
```
Comment (SparkYuan) [inline wordpress-cloud-rds/dev/platform.k:13]: ditto

File: wordpress-cloud-rds/dev/platform.k (hunk 1 above)
Comment (liu-hm19) [reply wordpress-cloud-rds/dev/platform.k:11]: The cloud rds in `dev/main.k` has defaulted to `AWS` RDS instance, and the attributes `subnetID` & `category` is not needed when provisioning an `AWS` RDS. If users want to use `Alicloud`, the related documentation has been updated in this [PR](https://github.com/KusionStack/kusionstack.io/pull/323) of kusionstack.io.

**Review by SparkYuan — APPROVED** [review]
lgtm

**Pull request merged**